[2023-06-16T12:42:41.249+0700] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: thethree_reqif_to_rst.build_rst manual__2023-06-16T05:42:16+00:00 [queued]>
[2023-06-16T12:42:41.256+0700] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: thethree_reqif_to_rst.build_rst manual__2023-06-16T05:42:16+00:00 [queued]>
[2023-06-16T12:42:41.257+0700] {taskinstance.py:1308} INFO - Starting attempt 1 of 0
[2023-06-16T12:42:41.268+0700] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): build_rst> on 2023-06-16 05:42:16+00:00
[2023-06-16T12:42:41.271+0700] {standard_task_runner.py:57} INFO - Started process 2816 to run task
[2023-06-16T12:42:41.279+0700] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'thethree_reqif_to_rst', 'build_rst', 'manual__2023-06-16T05:42:16+00:00', '--job-id', '304', '--raw', '--subdir', 'DAGS_FOLDER/mainflow.py', '--cfg-path', '/var/folders/v7/qsb_cvl92z92z6jf7g1jc0zr0000gn/T/tmpnw08tp84']
[2023-06-16T12:42:41.282+0700] {standard_task_runner.py:85} INFO - Job 304: Subtask build_rst
[2023-06-16T12:42:41.351+0700] {task_command.py:410} INFO - Running <TaskInstance: thethree_reqif_to_rst.build_rst manual__2023-06-16T05:42:16+00:00 [running]> on host macbook-pro.local
[2023-06-16T12:42:41.450+0700] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='The Three' AIRFLOW_CTX_DAG_ID='thethree_reqif_to_rst' AIRFLOW_CTX_TASK_ID='build_rst' AIRFLOW_CTX_EXECUTION_DATE='2023-06-16T05:42:16+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-16T05:42:16+00:00'
[2023-06-16T12:42:41.477+0700] {xcom.py:634} ERROR - Object of type bytes is not JSON serializable. If you are using pickle instead of JSON for XCom, then you need to enable pickle support for XCom in your airflow config or make sure to decorate your object with attr.
[2023-06-16T12:42:41.480+0700] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/airflow/utils/json.py", line 91, in default
    return serialize(o)
           ^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/airflow/serialization/serde.py", line 171, in serialize
    raise TypeError(f"cannot serialize object of type {cls}")
TypeError: cannot serialize object of type <class 'bytes'>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/lap14202/.airflow/dags/mainflow.py", line 43, in build_rst_task
    context['ti'].xcom_push(key='rst_file_content', value=rst_file_content)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/airflow/utils/session.py", line 76, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 2354, in xcom_push
    XCom.set(
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/airflow/utils/session.py", line 73, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/airflow/models/xcom.py", line 237, in set
    value = cls.serialize_value(
            ^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/airflow/models/xcom.py", line 632, in serialize_value
    return json.dumps(value, cls=XComEncoder).encode("UTF-8")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/json/__init__.py", line 238, in dumps
    **kw).encode(obj)
          ^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/airflow/utils/json.py", line 104, in encode
    return super().encode(o)
           ^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/json/encoder.py", line 200, in encode
    chunks = self.iterencode(o, _one_shot=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/json/encoder.py", line 258, in iterencode
    return _iterencode(o, 0)
           ^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/airflow/utils/json.py", line 93, in default
    return super().default(o)
           ^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/json/encoder.py", line 180, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type bytes is not JSON serializable
[2023-06-16T12:42:41.500+0700] {taskinstance.py:1345} INFO - Marking task as FAILED. dag_id=thethree_reqif_to_rst, task_id=build_rst, execution_date=20230616T054216, start_date=20230616T054241, end_date=20230616T054241
[2023-06-16T12:42:41.514+0700] {standard_task_runner.py:104} ERROR - Failed to execute job 304 for task build_rst (Object of type bytes is not JSON serializable; 2816)
[2023-06-16T12:42:41.545+0700] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-06-16T12:42:41.577+0700] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
